kafka/src/main/java/org/apache/apex/malhar/kafka/KafkaConsumerWrapper.java

```diff
@@ -74,7 +75,7 @@
 
   private static final Logger logger = LoggerFactory.getLogger(KafkaConsumerWrapper.class);
 
-  private boolean isAlive = false;
```

Comment:
In AbstractKafkaInputOperator operator, a new thread of consumerWrapper will be started: 
  @Override
  public void activate(Context.OperatorContext context)
  {
    consumerWrapper.start(isIdempotent());
  }

 which in fact created a thread pool to execute
// create thread pool for consumer threads
    kafkaConsumerExecutor = Executors.newCachedThreadPool(
      new ThreadFactoryBuilder().setNameFormat("kafka-consumer-%d").build());

  @Override
  public void deactivate()
  {
    consumerWrapper.stop();
  }